content/en/contribute/code/conf/development.md

```diff
@@ -0,0 +1,51 @@
+
+# Executing your local branch
+1. Clone the project locally
+2. Make changes to cht-conf or checkout a branch for testing
+3. Test changes
+    1. To test CLI changes locally you can run `node <project_dir>/src/bin/index.js`. This will run as if you installed via npm.
+    2. To test changes that are imported in code run `npm install <project_dir>` to use the local version of cht-conf.
+
+# Releasing
+1. Create a pull request with prep for the new release.
+2. Get the pull request reviewed and approved.
+3. When doing the squash and merge, make sure that your commit message is clear and readable and follows the strict format described in the commit format section below. If the commit message does not comply, automatic release will fail.
+4. In case you are planning to merge the pull request with a merge commit, make sure that every commit in your branch respects the format. 
+
+## Commit format
+The commit format should follow this [conventional-changelog angular preset](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular). Examples are provided below.
+
+| Type        | Example commit message                                                                              | Release type |
+|-------------|-----------------------------------------------------------------------------------------------------|--------------|
+| Bug fixes   | fix(#123): infinite spinner when clicking contacts tab twice                                        | patch        |
+| Performance | perf(#789): lazily loaded angular modules                                                           | patch        |
+| Features    | feat(#456): add home tab                                                                            | minor        |
+| Non-code    | chore(#123): update README                                                                          | none         |
+| Breaking    | perf(#2): remove reporting rates feature <br/> BREAKING CHANGE: reporting rates no longer supported | major        |
+
+## Releasing betas
+1. Checkout the default branch, for example `main`
+2. Run `npm version --no-git-tag-version <major>.<minor>.<patch>-beta.1`. This will only update the versions in `package.json` and `package-lock.json`. It will not create a git tag and not create an associated commit.
+3. Run `npm publish --tag beta`. This will publish your beta tag to npm's beta channel.
+
+To install from the beta channel, run `npm install cht-conf@beta`.
+
+# Build status
+Builds brought to you courtesy of GitHub actions.
```

Comment:
Do you have more information on what Build status is?

Reply:
Sorry, I don’t :face_with_diagonal_mouth:

Reply:
Okay, we can leave it as is and add on later.